README.md

```diff
@@ -22,7 +22,7 @@ console.log(results);
 ```
 
 ```js
-// Your answer...
+The syntax "AuthorModel", that would not be the correct name.  If you are trying to find a document with the value "Bob" for the key "name" in an author collection.
```

Comment:
The issue is that the code that we want to happen after the Mongoose query is not wrapped in a callback or promise. Mongoose queries are asynchronous.